parser/parser.y

```diff
@@ -4527,12 +4528,25 @@ ShowTableAliasOpt:
 	}
 
 FlushStmt:
-	"FLUSH" NoWriteToBinLogAliasOpt TableOrTables TableNameListOpt WithReadLockOpt
+	"FLUSH" NoWriteToBinLogAliasOpt FlushOption
+	{
+		tmp := $3
+		if x, ok := tmp.(*ast.FlushTableStmt); ok {
```

Comment:
Because it's written in C, C has pointer, and pointer is weak typed, so it's OK to use `lex->no_write_to_binlog`
Go type is more strict, type assert must be used for `interface{}` , `tmp.NoWriteToBinLog` doesn't work here.

@shenli